### PR TITLE
Merging rafaelvfalc/asperathos-monitor multi-cluster branch in ufcg/asperathos-monitor master branch

### PR DESCRIPTION
**Pull request by rafaelvfalc**
* Adding the functionality of add/delete/active a specific cluster in a Asperathos instance execution time.
* Updating documentation with new endpoints.
* Solving issue #7 
* Modifications of the unit tests of the component

### CODE REVIEW

**Comment by rafaelvfalc** [comment]
The PR is able to 'Squash and merge' with the master branch. It is not the best approach? In this way all four commits will be combined in one commit in the base branch.

File: monitor/service/api/v10.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 import ConfigParser
 import os
+import filecmp
+import shutil
```
Comment (clenimar) [inline monitor/service/api/v10.py]: you don't seem to be using these libs.

File: monitor/tests/unit/plugins/test_plugin.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 class TestKubeJobs(unittest.TestCase):
 
-        
+
```
Comment (clenimar) [inline monitor/tests/unit/plugins/test_plugin.py]: unrelated.

File: monitor/service/api/__init__.py [hunk 3]
```diff
@@ -44,10 +44,18 @@
         mesos_username = config.get('spark_mesos', 'mesos_username')
 
     if 'external_api' in plugins:
+
+        # Setting default value
+        k8s_manifest = "./data/conf"
```
Comment (clenimar) [inline monitor/service/api/__init__.py]: define in a constant to make it easier for future contributors/users.

File: monitor/service/api/v10.py (hunk 1 above)
Comment (rafaelvfalc) [reply monitor/service/api/v10.py]: Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d

File: monitor/tests/unit/plugins/test_plugin.py (hunk 2 above)
Comment (rafaelvfalc) [reply monitor/tests/unit/plugins/test_plugin.py]: Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d

File: monitor/service/api/__init__.py (hunk 3 above)
Comment (rafaelvfalc) [reply monitor/service/api/__init__.py]: Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d
